feature-libs/product-configurator/rulebased/core/state/reducers/index.ts

```diff
@@ -7,8 +7,10 @@ import { configuratorReducer } from './configurator.reducer';
 
 export function getConfiguratorReducers(): ActionReducerMap<ConfiguratorState> {
   return {
+    // @ts-ignore product-configurators
     configurations: StateUtils.entityProcessesLoaderReducer<Configurator.Configuration>(
       CONFIGURATOR_DATA,
+      // @ts-ignore product-configurators
```

Comment:
Can we create a ticket for it and reference as TODO(#xxxx) instead of `product-configurators`? Now it's not clear what `product-configurators` mean in this place.

Reply:
Yes, agree. I had this comment only to ease the discussion on remaining issues with you, not knowing what we finally accept as TODO